bbp/modules/default.nix

```diff
@@ -86,13 +86,13 @@ let
             conflicts = conflicts-modules;
         };
 
-        syntool = pkgs.envModuleGen rec {
-            name = "syn-tool";
+        synapsetool = pkgs.envModuleGen rec {
```

Comment:
Indeed this module is not testing by continuous-integration (and although not deployed).
If you want it to be, you have to reference the `synapsetool` module in `all` symbol below.

Reply:
Oops, actually the old one was already there as part of hpc_base, which had to be renamed. Should be ok now